src/scrape_up/goodreads/books.py

```diff
@@ -0,0 +1,187 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
```

Comment:
Hey @brohithkr | `ChromeDriverManager` does not work with the packages on all the devices, it requires installation of the chrome driver. 
Many of our modules are facing the same issue, try to look for an alternative.

Reply:
Give it a try

Reply:
The problem is, apart from download it needs to be set to the path in system's environment variables.